Packs/MalwareInvestigationAndResponse/ReleaseNotes/1_0_1.md

```diff
@@ -0,0 +1,6 @@
+
+#### Incident Fields
+##### New: **Malware Investigation Summary**
+- %%UPDATE_RN%% (Available from Cortex XSOAR 6.5.0).
```

Comment:
This should be removed ( or add a description at least).

Reply:
Done